modules/microshift-adding-service-to-blueprint.adoc

```diff
@@ -21,13 +26,48 @@ groups = []
 
 [[packages]]
 name = "microshift"
-version = "4.13.0-1"
+version = "*"
 
 [[packages]]
 name = "microshift-greenboot" <1>
-version = "4.13.0-1"
+version = "*"
 
 [customizations.services]
 enabled = ["microshift"]
+EOF
 ----
 <1> Optional `microshift-greenboot` RPM. For more information, read the "Greenboot health check" guide in the "Running Applications" section.
+
+[NOTE]
+====
+The wildcard `*` in the commands uses the latest {product-title} RPMs. If you need a specific version, substitute the wildcard for the version you want. For example, insert `4.13.1` to download the {product-title} 4.13.1 RPMs.
+====
+
+. Add the blueprint to the Image Builder by running the following command:
++
+[source,terminal]
+----
+$ sudo composer-cli blueprints push minimal-microshift.toml
+----
+
+.Verification
+
+. Verify the Image Builder configuration by running the following command:
++
+[source,terminal]
+----
+$ sudo composer-cli blueprints depsolve minimal-microshift
+----
+
+.Example output
++
+[source,terminal]
+----
+blueprint: minimal-microshift v0.0.1
+    microshift-greenboot-4.13.1-202305250827.p0.g4105d3b.assembly.4.13.1.el9.noarch
+    microshift-networking-4.13.1-202305250827.p0.g4105d3b.assembly.4.13.1.el9.x86_64
+    microshift-release-info-4.13.1-202305250827.p0.g4105d3b.assembly.4.13.1.el9.noarch
+    microshift-4.13.1-202305250827.p0.g4105d3b.assembly.4.13.1.el9.x86_64
+    microshift-selinux-4.13.1-202305250827.p0.g4105d3b.assembly.4.13.1.el9.noarch
```

Comment:
Now that we removed `| grep microshift`, we may want to add ... ... before and after the package list because microshift is only a subset

Reply:
@dhellmann Is there a reason the user would want all the information versus just microshift? Wondering if having everything is useful or creating noise.

Reply:
The output tells the user the actual versions of everything in the image, including any other packages the user has added to their blueprint.

Reply:
I clarified the first step using `| grep microshift`, then added an optional step after for a user who wants the whole enchilada.